### DISCUSSION TITLE
App Store Connect app pricing update broke CI that use Fastlane

### DISCUSSION THREAD

**Discussion by mregnauld** [codemagic-ci-cd/codemagic-docs#1519] [Q&A]
For the past few weeks, I can't deploy my iOS app anymore, as you can see in that workflow: `642187d0f144453321f8e488`.

Here is the error message:

```
The request cannot be fulfilled because of the state of another resource. - The resource 'appPrices' cannot be viewed, created or updated. Please view and create 'manualPrices' using the resource 'appPriceSchedules'. (Spaceship::UnexpectedResponse)
```

Apparently, Apple made some recent changes about app pricing that broke the CI made with Fastlane, as you can see [here](https://github.com/fastlane/fastlane/issues/21125).

It looks like there is a workaround, and I wonder if you could implement it on your end so my CI could work again.

Thanks.

**Comment by icarusdust** [comment]
Hi @mregnauld, if you are working with Fastlane integration, then I would assume that you need to adjust these changes in your Fastlane settings. If these Apple breaking changes broke your Fastlane pipeline, then it should break it locally as well.
I can further check it if you can provide another build id as this one is not found anymore.

**Comment by mregnauld** [comment]
Well, I just tried again and it's working now.
Thanks!
